### PR TITLE
Refactor to use graphql-tools

### PR DESCRIPTION
**Pull request by kutyel**
I recently refactored my first GraphQL endpoint to use `graphql-tools` and I ❤️  it, hope you too 😉

### CODE REVIEW

**Comment by SaraVieira** [comment]
Oh my god ! This is awesome ! I will download the repo after lunch and test it out 🎉

File: src/server/index.js [hunk 1]
```diff
@@ -1,95 +1,71 @@
...
+    # My GitHub name
+    github: String!
+    # Am I employed?
+    employed: Boolean!
+    jobs: [Job]
+    repos: [Repo]
+    talks: [Talk]
+    projects: [Project]
+  }
+  ${Job.concat(Project, Repo, Talk)}
```
Comment (kutyel) [inline src/server/index.js]: Maybe this line could be improved, but I found it funny that you can concatenate schemas like this 🤣

**Comment by kutyel** [comment]
If you want I can remove the `package-lock.json` file to actually show that is less code 😜 🤣

**Comment by SaraVieira** [comment]
No need and I said I would watch this today and I failed 😑

I'll run it tomorrow! I'm flying back to portugal  😎

It should be ready to go though  🍕

THANK YOU

**Comment by kutyel** [comment]
No problem, we all have busy lifes! 😉

**Comment by kutyel** [comment]
Managed to make prettier use standard back, I only had to touch every single file in the project 😅  sorry about it! 🤣

**Comment by SaraVieira** [comment]
Merged into another branch to be easier for me to see ALLL the things 😝

**Pull request merged**